rootfs/etc/nginx/lua/metric_emitter.lua

```diff
@@ -0,0 +1,50 @@
+local socket = ngx.socket.udp
+local cjson = require('cjson')
+local assert = assert
+
+local _M = {
+  queue = {}
+}
+
+local function flush_queue()
+  local s = assert(socket())
+  assert(s:setpeername("127.0.0.1", 8000))
+  for _, v in ipairs(_M.queue) do
+    assert(s:send(v))
+  end
+  _M.queue = {}
+  assert(s:close())
+end
+
+function _M.call()
+  -- Create JSON Metrics Payload  --
+  local jsonPayload = cjson.encode({
+    host                 = ngx.var.host                             or "-",
+    status               = ngx.var.status                           or "-",
+    remoteAddr           = ngx.var.realip_remote_addr               or "-",
```

Comment:
This is a bit confusing. `ngx.var.realip_remote_addr` is the original client IP, but `remoteAddr` indicates client IP (obtained from `X-Frowarded-For` if set). I'd say `ngx.var.remote_addr` is more valuable since it will be the true client IP.

Maybe have both?

Reply:
Good point. I'll add both to avoid confusion.